cmd/cmd.go

```diff
@@ -36,15 +36,32 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 
 	var spinner *Spinner
 
-	request := api.CreateRequest{Name: args[0], Path: filename}
-	fn := func(resp api.CreateProgress) error {
-		if spinner != nil {
-			spinner.Stop()
-		}
+	var currentDigest string
+	var bar *progressbar.ProgressBar
 
-		spinner = NewSpinner(resp.Status)
-		go spinner.Spin(100 * time.Millisecond)
+	request := api.CreateRequest{Name: args[0], Path: filename}
+	fn := func(resp api.ProgressResponse) error {
```

Comment:
not important for this change, but we should probably factor this out at some point for each of pull/push/create functions.